client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -215,7 +215,7 @@ class DomainDetailsForm extends PureComponent {
 			return [];
 		}
 
-		return cartItems.hasTld( this.props.cart, 'fr' ) ? [ 'fr' ] : [];
+		return cartItems.getTlds( this.props.cart );
```

Comment:
Side effect here: possibly will get multiple values back now.

That's good down the road but we should make sure we don't explode in the interim.